src/components/HeaderWithLanguage/HeaderActions.jsx

```diff
@@ -86,7 +92,7 @@ const HeaderActions = ({
           emptyLanguages={emptyLanguages}
           editUrl={editUrl}
         />
-        {articleType &&
+        {(articleType || conceptContent) &&
```

Comment:
Er det en bedre måte å sjekke om noe er concept/ en forklaring?

Reply:
Går an å bruke type propen:
```suggestion
        {translatableTypes.includes(type) &&
```